prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```

Comment:
Here as well.

Reply:
This one too.
![prep-css-flex-wrap-corrected](https://user-images.githubusercontent.com/109567499/183233766-9cc0e737-39fd-4746-9188-e6cab14433d3.PNG)